docker-entrypoint.sh

```diff
@@ -1,6 +1,12 @@
 #!/usr/bin/env sh
 set -e
 
+device_dir=$(ls /dev/longhorn/)
```

Comment:
This is dangerous if the node has more than one volume. Is there a way to figure out what's the PVC name then use it as the file name?

Reply:
I think we can to get the latest block device, it should be the one just created for this Job.